pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -1539,6 +1542,29 @@ func isConflictWithWellKnownPort(incoming, existing protocol.Instance, conflict
 	return true
 }
 
+func mergeListenerFilters(a, b []*listener.ListenerFilter) []*listener.ListenerFilter {
```

Comment:
I think the target of this function is  to return a union b, if so the implement is not correct

Reply:
Why is it not correct?

Reply:
Can you elaborate?

Reply:
I mean you just return the combination of a and b  for `TLSInspector` and `HTTPInspector`.

So how about b contains another listener filter that a does not contain?